faux-mgs/src/main.rs

```diff
@@ -857,11 +879,55 @@ async fn run_command(
                             .join(":")
                     ));
                     lines.push(format!("power state: {:?}", state.power_state));
-                    // TODO: pretty print RoT state?
-                    lines.push(format!("RoT state: {:?}", state.rot));
+                    match state.rot {
+                        Ok(rotstate) => {
+                            lines.push(format!(
+                                "{}",
+                                &RotStateV2Display(&rotstate)
```

Comment:
Can we put `.display()` helper methods on the relevant types so a caller doesn't have to construct this by hand? E.g.,

```rust
lines.push(format!("{}", rotstate.display()));
```

Reply:
This is only for human convenience in working with FWIDs. I could take this stuff out and add it later if it is still useful. It's a bunch of code with limited value since I have since worked out how to deal with  this in a jq script.

Reply:
I'm not sure that's relevant to my comment? :sweat_smile: We still have all this code, and it would be nicer to have `.display()` methods than needing to construct the adapters by hand.